eng/packages/http-client-csharp/generator/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsClientLibrary>false</IsClientLibrary>
+  </PropertyGroup>
+  <!--
+    Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
+  -->
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
```

Comment:
I think this points to the correct one since build works.
And there is no `Directory.Build.props` in eng, only `Directory.Build.Common.props`, which is used in the root `Directory.Build.props`.
So, I think this points to the root `Directory.Build.props`.